scheduler/src/cook/mesos/task.clj

```diff
@@ -140,9 +140,9 @@
 
 (defn TaskAssignmentResult->task-metadata
   "Organizes the info Fenzo has already told us about the task we need to run"
-  [db framework-id mesos-run-as-user ^TaskAssignmentResult task-result]
+  [db mesos-run-as-user ^TaskAssignmentResult task-result]
   (let [{:keys [job task-id] :as task-request} (.getRequest task-result)]
-    (merge (job->task-metadata db framework-id mesos-run-as-user job task-id)
+    (merge (job->task-metadata db @cook.config/framework-id-atom mesos-run-as-user job task-id)
```

Comment:
I kept this here. Long term when cook schedules over multiple domains, it can vary on a per-offer basis in the fenzo cycle. 

Longer term, this will be replaced with a function from the Fenzo result (indicating the scheduling domain matched) that will store the selected scheduling domain into the task structure (and subsequently use that in order to launch it in the right spot.).

Today, however, we have a single scheduling domain --- a single mesos cluster, so just stuff that in here.

we'll encode the domain in fenzo.

Reply:
Can we put a code comment reflecting what you said here?

Reply:
Done.